quickstart/docker/all-in-one/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM debian:bookworm-slim
 
-COPY ./build/ziti /usr/local/bin/
+ARG ARTIFACTS_DIR=./build
```

Comment:
This container image is much newer than the ziti-cli image and is needed because it can run the ziti binary that was built on the Docker host, which typically has a much newer glibc.

Put another way, if we use the ziti-cli image instead of this new Dockerfile, then we also must build the ziti binary in that image (or one with the same vintage of glibc), which is slower and impedes the developer who is using this all-in-one Compose project to run the locally-built ziti binary.